hooks/houdini/collector.py

```diff
@@ -121,6 +120,25 @@ def process_current_session(self, settings, parent_item):
         # Return the list of items
         return items
 
+    def add_items(self, items, settings, session_item):
+        """
+        For each item in houdini_sgtk_outputs set respective node_collected to False and add to items
+        eg for alembic node:
+            _alembic_nodes_collected = False
+            items.extend(collect_tk_alembicnodes(settings, session_item))
+        :param items: list with collected session items
+        :param settings: Configured settings for this collector
+        :param session_item: Current houdini session
+
+        :return: Item list with collected nodes
+        """
+        for node in self.houdini_sgtk_outputs[hou.ropNodeTypeCategory()]:
+            setattr(self, "_{}_nodes_collected".format(node), False)
+            collect_method = getattr(self, "collect_tk_{}nodes".format(node))
+            items.extend(collect_method(settings, session_item))
+
+        return items
```

Comment:
`add_items()` is too generic of a name. You could rename this something like `collect_sgtk_node_outputs` though I would just move this block to the beginning of `collect_node_outputs()`.